### PR TITLE
gamfantasy's proposal

### PR DESCRIPTION
**Pull request by wqwqwq219**
please review our proposal

### CODE REVIEW

**Comment by guo** [comment]
Thanks for the first proposal!

File: oep-proposals/open-proposal-gamefantasy.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+### Value
+---
+That will be the best choice for blockchain online games in this way. As mentioned above,
+An excellent game can referral traffic by taken payment, and then increase transaction volume and token usage. Games can only focus on the entertainment and provides the best experience for player. But during the process, the value of token will be increase more. 
+Taking a step back, it will not any other risks if this method cannot be accepted by market.
+The main difficulty of this project is to make our games run normally in various web environments and to recharge with the blockchain normally. However, I believe that there is technical support for the project, at least there is no problem in the interaction of the chain.
+
+### Deliverables
+---
+The game will deliver html5casual games mechanisms running within the page. Of course, make sure the game can run well is our responsibility. At least 80% of the bugs are fixed before the officially online, but it does not affect the player's experience of the entire game. Players can recharge and pay within the game by iotx or vita.
```
Comment (raullenchai) [inline oep-proposals/open-proposal-gamefantasy.md:22]: Please detail the plan for building this game. Will this be a card game or something else, and how exactly IOTX/VITA or IoTeX blockchain will be used here?

File: oep-proposals/open-proposal-gamefantasy.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+### Project Title:
```
Comment (raullenchai) [inline oep-proposals/open-proposal-gamefantasy.md:1]: Thanks for the application!

Comment (raullenchai) [reply oep-proposals/open-proposal-gamefantasy.md:1]: Pls combine lines 1, 2 and 3 to be: 
# Open Grant Proposal: `Cats Around The World`

File: oep-proposals/open-proposal-gamefantasy.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+---
+#### Name of Project:cats around the world
+#### Proposal Category: cat factory
```
Comment (raullenchai) [inline oep-proposals/open-proposal-gamefantasy.md:4]: Dapp

File: oep-proposals/open-proposal-gamefantasy.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+#### Proposer:wqwqwq219
+#### Do you agree to open source all work you do on behalf of this RFP?: No
```
Comment (raullenchai) [inline oep-proposals/open-proposal-gamefantasy.md:6]: Should think at least open the contract part of the game?

**Comment by wqwqwq219** [comment]
we update our proposal,please review it

**Comment by raullenchai** [comment]
Thanks for the proposal! The team has reviewed this proposal on 6/5 and here is the feedback:

We like the idea to make IOTX part of a game. We will accept this proposal if you could elaborate more about what type of this game is and how IOTX will be used in the game, e.g., having some screenshots of the game or design of the entire system/product would be really helpful!

From core-dev

**Comment by wqwqwq219** [comment]
we update our proposal,add the game demo link.please review it!

**Comment by raullenchai** [comment]
Congratulations!

**Pull request merged**